src/olmo_core/data/mixes/__init__.py

```diff
@@ -7,10 +7,19 @@
 
 from ..tokenizer import TokenizerName
 
-__all__ = ["DataMix"]
+__all__ = ["BaseDataMix", "DataMix"]
 
 
-class DataMix(StrEnum):
+class BaseDataMix(StrEnum):
+    """
+    Base class for enumeration of data mixes.
+    """
+
+    def build(self, base_dir: str, tokenizer: TokenizerName) -> Tuple[List[str], List[str]]:
```

Comment:
Can you move the docstring for this method below to here?

Reply:
Done